plugin/src/shared/sc/plugin2021/util/MoveMistake.kt

```diff
@@ -0,0 +1,46 @@
+package sc.plugin2021.util
+
+import sc.plugin2021.Color
+import sc.plugin2021.Move
+import sc.plugin2021.PieceShape
+
+/** Wird optional bei Validierung von Zügen zurückgegeben, falls ein Zug laut Regeln verboten ist. */
+sealed class MoveMistake(
+        val color: Color
+)
```

Comment:
```suggestion
sealed class MoveMistake(val color: Color)
```

Reply:
Applied in e5bd1117ffab9717da989fbf883d59b1dcc5015d